lib/vrf.c

```diff
@@ -1006,7 +1006,7 @@ int vrf_bind(vrf_id_t vrf_id, int fd, const char *name)
 	 * case ifname = vrf in netns mode => return
 	 */
 	ifp = if_lookup_by_name(name, vrf_id);
-	if (!ifp)
+	if (!ifp || ifp->ifindex == 0)
```

Comment:
I suggest using `IFINDEX_INTERNAL` for 0.

Reply:
Done